tests/GetAccountsBalancesTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
```

Comment:
+ strict types?

Reply:
In version v1.x we still have `~5.3` in composer.json, i am not sure but static analysis can have problem with strict type on old versions.

I can add strict type while i will merging v1.x into v2.x.

Reply:
That's right, but tests run from PHP 7.2 (I don't know, if this version supports strict types). It's not so much important, so it's up to you.